cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguageFrontend.kt

```diff
@@ -63,6 +64,7 @@ import kotlin.math.min
  * a value (on the first assignment).
  */
 @RegisterExtraPass(PythonAddDeclarationsPass::class)
+@RegisterExtraPass(PythonUnreachableEOGPass::class)
```

Comment:
Do we want to have this pass on by default? Why isn't it part of `defaultPasses` (the non-Python version obviously) and then Python replaces it with this?
I'm just wondering if this is a future we only want to have active by default for Python?

Reply:
Well since this pass is massively slow (at the moment), we probably do not want it active per-default. So we can either keep this PR open as blocked or I will remove the register extra pass and if someone wants it, they can activate it.

Reply:
We won't activate this pass by default.